pkg/kritis/review/review_test.go

```diff
@@ -111,7 +111,134 @@ func TestHasValidAttestations(t *testing.T) {
 	}
 }
 
-func TestReview(t *testing.T) {
+func TestReviewGAP(t *testing.T) {
+	sec, pub := testutil.CreateSecret(t, "sec")
+	secFpr := sec.PgpKey.Fingerprint()
+	img := testutil.QualifiedImage
+	sig, err := util.CreateAttestationSignature(img, sec)
+	if err != nil {
+		t.Fatalf("unexpected error %v", err)
+	}
+
+	sMock := func(_, _ string) (*secrets.PGPSigningSecret, error) {
+		return sec, nil
+	}
+	validAtts := []metadata.PGPAttestation{{Signature: sig, KeyID: secFpr}}
+
+	invalidSig, err := util.CreateAttestationSignature(testutil.IntTestImage, sec)
+	if err != nil {
+		t.Fatalf("unexpected error %v", err)
+	}
+	invalidAtts := []metadata.PGPAttestation{{Signature: invalidSig, KeyID: secFpr}}
+
+	var gap = v1beta1.GenericAttestationPolicy{
```

Comment:
This is equivalent to gap :=

Reply:
Good point, updated here and in the ReviewISP test.